Makefile

```diff
@@ -0,0 +1,17 @@
+CXXFLAGS += -I include/mapbox/vector_tile --std=c++14 -Wall -Wextra -Werror 
```

Comment:
Ah, was using a local version of hpp-skel and hadn't updated. Looks like this is already fixes upstream. Thanks @springmeyer